trustly-alpha-merchant-backend/client/.well-known/apple-app-site-association

```diff
@@ -0,0 +1,26 @@
+{
+    "applinks": {
```

Comment:
Is this required to run the example backend? Or only to integrate with the alpha merchant apps?